triage/triaging.rst

```diff
@@ -7,32 +7,24 @@ Triaging an Issue
 This section of the devguide documents the :ref:`issue tracker <tracker>` for
 users and developers.
 
-Contributors with the Triager role on the issue tracker can triage issues
-directly without any assistance.
-
 
 Checklist for Triaging
 ======================
 
-* Read the issue comment(s).
-* Review and set classification fields
-    - Title: should be concise with specifics which are helpful to someone
-      scanning a list of issue titles. (Optional, if possible) Add a
-      prefix at the start of the title to indicate the module, e.g. IDLE,
-      doc, or async.
-    - Type
-    - Stage
-    - Components: multiple items may be set
-    - Versions: set if known, leave blank if unsure. Multiple items may be set.
-* Review and set process fields
-    - Status
-    - Superseder
-    - Assignees
-    - Nosy List
-    - Priority
-    - Keywords
-* (Optional) Leave a brief comment about the proposed next action needed. If
-  there is a long message list, a summary can be very helpful.
+* Check that the title is concise with specifics which are helpful to
+  someone scanning a list of issue titles.
+* For pull request, ensure that the corresponding issue is added before
+  the title (``gh-NNNNN: ...``).
+* Read the initial message and the comments.
+* Set all the relevant :ref:`labels <gh-labels>`.
+* Where appropriate, you might set the "Assignees", "Reviewers", and
+  "Projects" fields.
+* You might also leave a brief comment about the proposed next action needed.
+  If there is a long message list, a summary can be very helpful.
+* Finally, you can set the :gh-label:`triaged` label (unless you want other triagers
+  to take a look).
```

Comment:
This is so far the first and only documentation we have about the "triaged" label, so let me try to understand what it's for.  The idea is that if a triager **does not want** other triagers to look at the issue, they put a triaged label?

Why would a triager not want other triagers to look at the issue?

Is it then offensive for another triager or core dev to change the labels on the issue (undermining a decision that another triager marked at 'final')?

This sounds to me like the opposite of how we work.  It's the equivalent of a label that a no-committer can put on a PR to say "this has been reviewed. I do not want other non-commiters to review it".  We don't have this label. We have the "awaiting core review" label which indicates that a non-committer has reviewed, but it does not dis-invite further reviews from non-committers.

Reply:
I see now that if I hover over the label it says "The issue has been accepted as valid by a triager". Which is quite different from what is implied by this doc. 

I don't think validity is something we expect triagers to decide about (when I proposed a "valid" label I tried to be clear that it is to be used by subject matter experts, not triagers). If you think that this is for triagers to decide, then this doc should explain how a triager is supposed to make that decision. If you don't think that, then the tooltip on the "triaged" label needs to change to whatever it is that this label means.

Reply:
I don't remember the exact timeline, but I wrote that sentence on Aug 6 and I might have added the label description on GitHub later after more discussions.

However it seems there is still no clear consensus on how it should be used.  Are you at the sprints?  Perhaps it could be discussed there.

Regarding this specific PR, it is mostly about updating the checklist, not documenting the label itself.  I can remove or change the sentence, and possibly re-add/update it again in next iteration.  I also have another incoming PR (#930) where I can document it more extensively.

Reply:
What meaning did the Steering Council intend when they approved the new term?

I agree with deferring the Triaged sentence to get on with the improved list.

Reply:
See https://discuss.python.org/t/improve-communication-with-contributors-on-the-issue-tracker-and-prs-language-summit-follow-up/15812/30, in particular: 

> The triaged label would signify that the process is done (for now, at least) – and since the issue wasn’t closed, it’s “valid” and waiting for a core dev.

Reply:
> Are you at the sprints? Perhaps it could be discussed there.

I'm online, but I don't feel able to contribute on this matter beyond commenting on whether I think the documentation is consistent and clear.

Reply:
> The idea is that if a triager does not want other triagers to look at the issue, they put a triaged label?

No, the idea is that if a triager thinks there is nothing more a triager can do, they mark the issue so others don't need to re-read it.

> Is it then offensive for another triager or core dev to change the labels on the issue (undermining a decision that another triager marked as 'final')?

That would be like any other disagreement. It could be done in an offensive way, but hopefully won't.

> this has been reviewed. I do not want other non-commiters to review it

I'd word it as “This has been reviewed. Other non-commiters are free to re-review it, but I believe their time would be better spent elsewhere.”

*(I don't speak for the entire Steering Council here.)*

Reply:
> The triaged label would signify that the process is done (for now, at least)

Does “for now, at least” mean that the triaged label will be removed by a bot after a certain amount of time? Or how else will it work long term?